### PR TITLE
4.4 execution plan summary

### PR DESCRIPTION
**Pull request by martin-neotech**
Added missing execution plan operators to the summary list.

Anti - https://neo4j.com/docs/cypher-manual/4.4/execution-plans/operators/#query-plan-anti
ExhaustiveLimit - https://neo4j.com/docs/cypher-manual/4.4/execution-plans/operators/#query-plan-exhaustive-limit
ShortestPath - https://neo4j.com/docs/cypher-manual/4.4/execution-plans/operators/#query-plan-shortest-path
TriadicBuild - https://neo4j.com/docs/cypher-manual/4.4/execution-plans/operators/#query-plan-triadic-build
TriadicFilter - https://neo4j.com/docs/cypher-manual/4.4/execution-plans/operators/#query-plan-triadic-filter

### CODE REVIEW

**Comment by martin-neotech** [comment]
@hedengran please review and check if the `leaf`, `updating`, or `considerations` column need information also to the missing operators in the list. Thanks.

File: cypher/cypher-docs/src/docs/dev/execution-plans-operator-summary.asciidoc [hunk 1]
```diff
@@ -208,6 +214,14 @@ Tests for the absence of a pattern predicate.
 |
 |
 
+| <<query-plan-exhaustive-limit,ExhaustiveLimit>>
+a|
+The `ExhaustiveLimit` operator is similar to the `Limit` operator, but will always exhaust the input.
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/docs/dev/execution-plans-operator-summary.asciidoc]:
```suggestion
The `ExhaustiveLimit` operator is similar to the `Limit` operator, but always exhausts the input.
```

File: cypher/cypher-docs/src/docs/dev/execution-plans-operator-summary.asciidoc [hunk 2]
```diff
@@ -368,34 +382,34 @@ Tests for the presence of a pattern predicate in queries containing multiple pat
...
+| <<query-plan-optional-expand-all,OptionalExpand(All)>>
+| Traverses relationships from a given node, producing a single row with the relationship and end node set to `null` if the predicates are not fulfilled.
 |
 |
 |
 
-| <<query-plan-optional,Optional>>
-| Yields a single row with all columns set to `null` if no data is returned by its source.
+| <<query-plan-optional-expand-into,OptionalExpand(Into)>>
+| Traverses all relationships between two nodes, producing a single row with the relationship and end node set to `null` if no matching relationships are found (the start node will be the node with the smallest degree).
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/docs/dev/execution-plans-operator-summary.asciidoc]:
```suggestion
| Traverses all relationships between two nodes, producing a single row with the relationship and end node set to `null` if no matching relationships are found (the start node is the node with the smallest degree).
```

**Review by AlexicaWright — COMMENTED** [review]
Small editorial comments. I haven't built it since there are no artifacts on TC yet.

File: cypher/cypher-docs/src/docs/dev/execution-plans-operator-summary.asciidoc [hunk 3]
```diff
@@ -70,20 +76,20 @@ Tests for the absence of a pattern predicate.
...
+| <<query-plan-create-index,CreateIndex>>
+| Creates an index for either nodes or relationships.
 |
 | label:yes[]
 |
 
-| <<query-plan-create-nodes---relationships,Create>>
-| Creates nodes and relationships.
+| <<query-plan-create-node-key-constraint,CreateNodeKeyConstraint>>
+| Creates a node key constraint on a set of properties for all nodes having a certain label.
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/docs/dev/execution-plans-operator-summary.asciidoc]:
```suggestion
| Creates a node key constraint on a set of properties for all nodes with a certain label.
```

**Review by AlexicaWright — COMMENTED** [review]
Found one more thing, but I've also built it now and everything else looks good! Thank you @martin-neotech !

**Review by AlexicaWright — APPROVED** [review]
LGTM 👍 ! Well done @martin-neotech , thank you!

**Review by blouerat — APPROVED** [review]

**Pull request merged**